topics/genome-annotation/tutorials/introduction/slides.html

```diff
@@ -252,6 +275,14 @@
 
 ---
 
+### Helixer
+
+- Annotation based on the development and use of a cross-species deep learning model
+- Ab-initio annotation of core genes between species of large eukaryotic genomes
+- Model capable of differentiating between different intergenic regions
```

Comment:
This slide would deserve to be reworked: the important message is that it's a new and different approach, that it can predict genes without evidences, it uses GPU and runs fast